include/odp/api/spec/cpu.h

```diff
@@ -269,6 +269,15 @@ void odp_prefetch_strm_l1(const void *addr);
  */
 void odp_prefetch_store_strm_l1(const void *addr);
 
+/**
+ * Prefetch into L1 instruction cache
+ *
+ * Hints CPU to prefetch application code into L1 instruction cache.
```

Comment:
Maybe the "application code" could be just "code".

Reply:
Fixed in v4.